### ISSUE TITLE
Binding async methods

### ISSUE THREAD

**Issue by mensch272** [arcticfox1919/LuaDardo#9]
Just wondering if this library has the ability to bind and call async methods in Lua.

**Comment by alonohana627** [comment]
You need to manually do it but in general - you are able. I will post a code snippet soon.

**Comment by maks** [comment]
I don't think this is currently possible per my [comment](https://github.com/arcticfox1919/LuaDardo/issues/7#issuecomment-1655033851) in issue #7 but if you know a workaround @alonohana627 I would be very keen to find out about it.

Actually I think this should really be marked as a duplicate of #7.
